example/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">RatingBarExample</string>
+    <string name="random_star_size">Random Star Size</string>
+    <string name="random_star_count">Random Star Count</string>
+    <string name="indicatorswitch">IndicatorSwitch</string>
```

Comment:
@raigex consistent resource names -> use an underscore to separate `indicator` and `switch`